src/NBomber/DomainServices/Reporting/Report.fs

```diff
@@ -21,7 +21,7 @@ type ReportResult = {
 let build (dep: Dependency, stats: GlobalStats,
            assertResults: Result<Assertion,DomainError>[]) =
     { TxtReport = TxtReport.print(stats)
-      HtmlReport = HtmlReport.print(dep, stats)
+      HtmlReport = HtmlReport.print(dep, stats, assertResults)

```

Comment:
I guess you could filter assertResults here and pass only DomainError[] or maybe even on level up